client/src/components/app/App.css

```diff
@@ -0,0 +1,3 @@
+#root > div {
```

Comment:
I don't understand this css used

Reply:
It was to give necessary margins to the container div created for the routes, which routes shouldn’t have to handle individually (for consistency) but removing for now.